src/CitMovie.Data/RepositoryInterfaces/IBookmarkRepository.cs

```diff
@@ -1,40 +1,9 @@
-namespace CitMovie.Data;
```

Comment:
I would guess this shouldn't have been removed

Reply:
Hmmm ... Tested it everything works fine but no, that is not intentionel.